llarp/net/ip_range.hpp

```diff
@@ -24,6 +25,27 @@ namespace llarp
         : addr{std::move(address)}, netmask_bits{std::move(netmask)}
     {}
 
+    explicit IPRange(std::string _range)
```

Comment:
i dont think this constructor needs to be `explicit`

Reply:
Single argument constructors should *always* be `explicit` by default and only made non-explicit if you are sure you want that.  Not being explicit means you want:
```C++
void somefunc(IPRange x) { ... }

std::string x = "...";
somefunc(x);
```
to be allowed.